test/rpc_test.py

```diff
@@ -962,13 +962,11 @@ def test_call_method_on_rref(self):
 
         self.assertEqual(result, sum(vals))
 
-    @requires_process_group_agent("PROCESS_GROUP rpc backend specific test, skip")
     @dist_init
     def test_get_default_rpc_timeout(self):
         timeout = rpc.get_rpc_timeout()
```

Comment:
See the discussion in https://github.com/pytorch/pytorch/pull/28926, we should directly add docstrings to the pybind method.

Reply:
@pritamdamania87 currently, `ProcessGroupAgent` is not exposed in the docs since I assumed that this is a private class and there's no reason for the user to directly access it (we provide higher level APIs for everything that needs to be done). If we wanted to move this doc to pybind, we'd have to expose documentation for `ProcessGroupAgent`, since `get_rpc_timeout` is a method on that class.

Reply:
Created issue #29628 for tracking thihs.

Reply:
Can we move this method from `RpcAgent` to `torch.distributed.rpc` and the implementation uses the default RpcAgent?